cli.py

```diff
@@ -87,5 +90,16 @@ def attr_upd():
                 await show_oven_menu(backend_selector, auth, args.said, session)
                 return
 
+        for bf_data in appliance_manager.beer_fridges:
+            if bf_data["SAID"]:
```

Comment:
This would always select the first fridge, even if the user has more than 1.
any reason to deviate from the way the other appliances' selection works? I would prefer to keep it consistent (passing the SAID as an argument)